tests/integration/src/test/resources/org/mule/test/integration/xml/xslt-functional-test-flow.xml

```diff
@@ -17,7 +17,7 @@
     <!--// START SNIPPET: transformer-config -->
     <mulexml:xslt-transformer name="xslt">
         <mulexml:xslt-text>
-            <xsl:stylesheet version="2.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+            <xsl:stylesheet version="2.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns="">
```

Comment:
I have confirmed that this "issue" is not the same behaviour as it is in Saxon 9.7.0-x. But at the same time this is they way it was working since Saxon 9.6 was upgraded. So, I would say keep this as it is in order to avoid breaking compatibility and during the upgrade for Saxon 9.7 in order to support XPath 3.1 write this breaking change in migration guideline.
What do you think?